src/test/resources/rest-api-spec/test/mapper_attachments/00_basic.yaml

```diff
@@ -10,5 +10,5 @@
     - do:
         nodes.info: {}
 
-    - match:  { nodes.$master.plugins.0.name: mapper-attachments  }
+    - match:  { nodes.$master.plugins.0.name: elasticsearch-mapper-attachments  }
```

Comment:
I don't think this should be necessary. Esplugin defaults to the project name, and gradle defaults the project name to the directory name. Instead of changing this, if you add  the name explicitly in build.gradle under the esplugin block, then it wont matter what directory the repo is checked out into.